src/app/public/modules/phone-field/fixtures/phone-field.component.fixture.html

```diff
@@ -0,0 +1,11 @@
+<div>
+  <sky-phone-field #phoneField>
+    <input
+      [(ngModel)]="modelValue"
```

Comment:
Style - Two-way binding should come after input bindings.

Reply:
Thanks. Fixed